### PR TITLE
Fixes DOC-1449

### PR DESCRIPTION
**Pull request by lamagnifica**
@stroilova @mulby @brianhw,  please comment on accuracy and if any further detail is needed.
@bradenmacdonald , FYI, only the effect of your https://github.com/edx/edx-platform/pull/6492 on "other" events is being addressed in this PR (library events to follow).
@mhoeber , @srpearce , @catong, this should go out as soon as the analytics team says it's ready.

### CODE REVIEW

**Comment by mulby** [comment]
:+1:

**Comment by mhoeber** [comment]
:+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -169,24 +169,31 @@ definitions of all events.
...
-value(s) from the ``user_api_usercoursetag`` table for the user. See
-:ref:`user_api_usercoursetag`.
+The following member fields are included to provide additional information
+when applicable for an event.
+
+* When included, ``course_user_tags`` contains a dictionary with the key(s)
+  and value(s) from the ``user_api_usercoursetag`` table for the user. See
+  :ref:`user_api_usercoursetag`.
+* When included for a server event, ``module`` contains a dictionary with the
+  component ``display_name`` and the hashed ``usage_key`` of an xblock. These
```
Comment (stroilova) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: @lamagnifica , @brianhw  why is the usage_key "hashed" here? Isn't this just the usage_key / identifier of the xblock, stored without hashing? Also we've been asked what the 'usage_key' is.

Comment (mulby) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: ou, good catch. This should be just the usage_key.

**Comment by stroilova** [comment]
:+1:  Thank you!

**Pull request merged**